libnetdata/libjudy/src/JudyL/JudyLTablesGen.c

```diff
@@ -15,7 +15,7 @@
 // Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 // _________________
 
-// @(#) $Revision: 4.37 $ $Source: /judy/src/JudyCommon/JudyTables.c $
```

Comment:
Why do we want to change this?

Reply:
dont have to, but it makes it obvious this file is not 1:1 copy from judy source anymore

Reply:
Got it, IMHO it would be better to avoid touching unnecessary lines and rely on git for such info.

Reply:
I disagree on that somewhat the text in source and generated source file would say it was generated by version `4.37` but this is now not the same version.
I consider this however not important much so I can change that if you insist.